examples/Demo/FluentUI.Demo.Client/Documentation/GetStarted/Migration/MigrationFluentLabel.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Migrating FluentLabel
+route: /Migration/Label
+hidden: true
+---
+
+### New properties
+
+- `Size`, sets the label font size.
+- `Required`, sets required field styling by adding an asterisk 
+
+### Changed properties 🔃
+- `Weight`, now used to determine if the label text is shown regular or semibold
+
+### Removed properties💥
+
+- `Alignment`
+- `Color`
```

Comment:
Can we add `Color` and `CustomColor`? So the dev don't need to add a CSS style to do that (and we keep more compatibility with Lib v4)?

Same for `MarginBlock`

Reply:
No, not for the Label component. We do want to add those to the Text component